docs/document/i18n/ja/docusaurus-plugin-content-docs/current/guides/table-readonly-table-customization.md

```diff
@@ -94,7 +93,7 @@ const renderName = cellData => {
 ### 基本的な使い方
 
 #### ReadOnlyTable
-![readonly-table](assets/readonly-table.png)
+![readonly-table](/img/readonly-table.png)
 
 読み取り専用のテーブルを表示します。
```

Comment:
The blue background is not good. Please follow EN version style.
And I think no need border between "Show code" and source code itself.

Reply:
I have fixed